README.md

```diff
@@ -25,20 +24,20 @@ For the best experience using the Terraform Provider for Cloud Foundry, we recom
 
-🐞 - If you find a bug, feel free to create a [bug report](https://github.com/SAP/terraform-provider-cloudfoundry/issues/new?assignees=&labels=bug%2Cneeds-triage&projects=&template=bug_report.yml&title=%5BBUG%5D).
+🐞 - If you find a bug, feel free to create a [bug report](https://github.com/cloudfoundry/terraform-provider-cloudfoundry/issues/new?assignees=&labels=bug%2Cneeds-triage&projects=&template=bug_report.yml&title=%5BBUG%5D).
 
-💡 - If you have an idea for improvement or a feature request, please open a [feature request](https://github.com/SAP/terraform-provider-cloudfoundry/issues/new?assignees=&labels=enhancement%2Cneeds-triage&projects=&template=feature_request.yml&title=%5BFEATURE%5D).
+💡 - If you have an idea for improvement or a feature request, please open a [feature request](https://github.com/cloudfoundry/terraform-provider-cloudfoundry/issues/new?assignees=&labels=enhancement%2Cneeds-triage&projects=&template=feature_request.yml&title=%5BFEATURE%5D).
 
 ## Security / Disclosure
 
-If you find any bug that may be a security problem, please follow our instructions at [in our security policy](https://github.com/SAP/terraform-provider-cloudfoundry/security/policy) on how to report it. Please do not create GitHub issues for security-related doubts or problems.
+If you find any bug that may be a security problem, please follow our [instructions](https://www.cloudfoundry.org/security/) on how to report it. Please do not create GitHub issues for security-related doubts or problems.
 
 ## Code of Conduct
 
-Members, contributors, and leaders pledge to make participation in our community a harassment-free experience. By participating in this project, you agree to always abide by its [Code of Conduct](https://github.com/SAP/.github/blob/main/CODE_OF_CONDUCT.md).
+Members, contributors, and leaders pledge to make participation in our community a harassment-free experience. By participating in this project, you agree to always abide by its [Code of Conduct](https://www.cloudfoundry.org/wp-content/uploads/2015/09/CFF_Code_of_Conduct.pdf).
 
 ## Licensing
 
-Copyright 2024 SAP SE or an SAP affiliate company and `terraform-provider-cloudfoundry` contributors. See our [LICENSE](LICENSE) for copyright and license information. Detailed information, including third-party components and their licensing/copyright information, is available [via the REUSE tool](https://api.reuse.software/info/github.com/SAP/terraform-provider-cloudfoundry).
+Copyright 2024 SAP SE or an SAP affiliate company and `terraform-provider-cloudfoundry` contributors. See our [LICENSE](LICENSE) for copyright and license information. Detailed information, including third-party components and their licensing/copyright information, is available [via the REUSE tool](https://api.reuse.software/info/github.com/cloudfoundry/terraform-provider-cloudfoundry).
```

Comment:
Do we require Copyright SAP SE ? Can we check if this Licensing is necessary ?

Reply:
This is accepted. Checked with Beyhan

Reply:
The ones to be changed...Code of Conducted & Security is updated